tools/src/FileContents.cc

```diff
@@ -48,11 +49,31 @@ void printContents(const char* filename, const orc::ReaderOptions opts) {
 
 int main(int argc, char* argv[]) {
   if (argc < 2) {
-    std::cout << "Usage: file-contents <filename>\n";
+    std::cout << "Usage: file-contents <filename> "
+        << "[--columns=column1,column2,...]\n" ;
     return 1;
   }
   try {
+    const std::string COLUMNS_PREFIX = "--columns=";
+    std::list<int64_t> cols;
+
+    // Read command-line options
+    char *param, *value;
+    for (int i = 2; i < argc; i++) {
```

Comment:
You should also accept the more standard ordering of "file-contents --columns=1,2 filename.orc"